### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rozbeikonazar**

### CODE REVIEW

File: accounts/forms.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from django import forms
-from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
 from django.contrib.auth import get_user_model
 from .models import Profile
 # Create your forms here.
```
Comment (rrozb) [inline accounts/forms.py:5]: Remove those comment from all files.

Comment (rozbeikonazar) [reply accounts/forms.py]: Done

File: cart/views.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+def checkout(request):
+    items, order = get_order_items(request)
+    return render(request, 'cart/checkout.html', context={'items': items, 'order': order})
+
+def update_item(request):
+    data = json.loads(request.body)
+    product_id = data.get('productId')
+    action = data.get('action')
+    print(f"action: {action}, productId: {product_id}")
```
[the remark this answers is not in the document]
Comment (rozbeikonazar) [reply cart/views.py]: Done

File: accounts/views.py [hunk 3]
```diff
@@ -21,17 +23,19 @@ def register_request(request):
...
 		form = AuthenticationForm(request, data=request.POST)
 		if form.is_valid():
-			username = form.cleaned_data.get('username')
+			email = form.cleaned_data.get('username')
 			password = form.cleaned_data.get('password')
-			user = authenticate(username=username, password=password)
+			user = authenticate(username=email, password=password)
 			if user is not None:
 				login(request, user)
 				return redirect("products:product_list")
```
[the remark this answers is not in the document]
Comment (rrozb) [reply accounts/views.py:37]: not a good idea

File: cart/templates/cart/checkout.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+     <div class="row">
+		<div class="col-lg-6">
+			<div class="box-element" id="form-wrapper">
+				<form id="form">
+					<div id="user-info">
+						<div class="form-field">
+							<input required class="form-control" type="text" name="name" placeholder="Name..">
+						</div>
+						<div class="form-field">
+							<input required class="form-control" type="email" name="email" placeholder="Email..">
```
[the remark this answers is not in the document]
Comment (rrozb) [reply cart/templates/cart/checkout.html]: looks strange on github but source code is ok

**Review by rrozb — APPROVED** [review]

**Pull request merged**